.gitignore

```diff
@@ -48,8 +48,5 @@ crashlytics.properties
 crashlytics-build.properties
 fabric.properties
 
-# Windows exe files
-*.exe
-
 # Ignore Mac DS_Store files
 .DS_Store
```

Comment:
you pushed chromedriver again, resolve this

Reply:
I have to delete those 3 lines with *.exe?